powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehavior.java

```diff
@@ -902,7 +921,7 @@ private enum ProximityCheckResult {
         ERROR
     }
 
-    public record OperationListRequest(String userId, List<String> applications, Pageable pageable) {
+    public record OperationListRequest(String userId, List<String> applications, Optional<String> activationId, Pageable pageable) {
```

Comment:
Please do not use Optional as a method parameter or constructor. Internally it may be a nullable field and the return type of getter could be Optional.